site/docs/developer_documentation/ONV_path_manipulation.md

```diff
@@ -0,0 +1,212 @@
+
+## The key operators
+
+ONV path manipulations are described using a second quantization approach. GQCP's API sticks as closely to this as possible. To manipulate a path, we use the elementary second quantization operators to annihilate or create arcs that represent occupied orbitals. In an ONV path, a diagonal arc represents an occupied orbital, while a vertical arc represents an unoccupied orbital.  
+
+```C++
+/**
+*  Annihilate the diagonal arc that starts at the coordinate (q,n).
+* 
+*  @param q        the index of the orbital that should be annihilated
+*  @param n        the number of electrons in the ONV/path up to the orbital index q
+*/
+void ONVPath::annihilate(const size_t q, const size_t n) {
+
+    // During annihilation, we're removing an arc, so we'll have to update the current address by removing the corresponding arc weight.
+    this->m_address -= this->onv_basis.arcWeight(q, n);
+
+    // Update the next possible creation index. Since we're always constructing paths from the top-left to the bottom-right, we're only considering creation indices p > q.
+    this->p = q + 1;
+}
+
+/**
+*  Create the diagonal arc that starts at the coordinate (p, n).
+* 
+*  @param p        the index of the orbital that should be created
+*  @param n        the number of electrons in the ONV/path up to the orbital index q, prior to the creation
+*/
+void ONVPath::create(const size_t p, const size_t n) {
+
+    // During creation, we're adding an arc, so we'll have to update the current address by adding the corresponding arc weight.
+    this->m_address += this->onv_basis.arcWeight(p, n);
+}
+```
+
+The function `annihilate()` removes a diagonal arc, starting at the given coordinate in the ONV path. The function `create()` does the opposite. To understand this, lat's look at an example. 
+
+![init_path](/GQCP/img/ONVPath_10110.png)
+
+This path is the visual representation of ONV `10110`. There are three diagonal arcs (starting at [0, 0], [2, 1] and [3, 2]). These are the coordinates where `annihilate()` can remove an electron. Let's say you `annihilate(0, 0)`. The diagonal arc starting at [0, 0] will become a vertical arc, resulting in a new path:
+
+![firs_annihilate](/GQCP/img/ONVPath_00110_1.png)
+
+The ONV path is now 'open'. To 'close' it, the `create()` function can be used. In this particular case, the path can be closed by creating at coordinate [1, 1], resulting in a new ONV: `01110`.
+
+![firs_create](/GQCP/img/ONVPath_01110.png)
+
+We will come back to this later, when looking at a complete example, side-by-side with the source code. 
+
+Another important function is `leftTranslate()`.
```

Comment:
```suggestion
This is not the only way to close the path. A more involved way consists of shifting the following diagonals (occupied orbitals) to the left until a vertical arc is encountered to close the gap. This is the use of `leftTranslate()`.
```

Reply:
I think it is more clear to directly explain the use of `leftTranslate` instead of first showing the code.